api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -343,6 +344,11 @@ public WorkspaceResponse getWorkspace(String projectName, String workspaceName)
                 projectName, workspaceName, FIRECLOUD_GET_WORKSPACE_REQUIRED_FIELDS));
   }
 
+  @Override
```

Comment:
A purist might argue that I'm breaking a glass wall by introducing a dependency on a data model class into this service. Seems really convenient though, as I can never remember what the arguments are to this function.